### DISCUSSION TITLE
Proof of concept?

### DISCUSSION THREAD

**Discussion by NickDarvey** [nsacyber/HIRS#870] [Q&A] [answered]
I notice this has been around since at least 2018. Is it still considered a proof-of-concept and not recommended for production use as mentioned in [the wiki](https://github.com/nsacyber/HIRS/wiki)?

**Comment by iadgovuser26** [comment, marked as answer]
Correct. The Host Integrity at Runtime and Start-up project is still a [proof of concept](https://github.com/nsacyber/HIRS/wiki/FAQ#is-the-aca-intended-for-use-in-an-operational-system).
